website/docs/cloud-docs/workspaces/settings/vcs.mdx

```diff
@@ -71,13 +71,21 @@ This option instructs Terraform to begin new runs only for changes that have a s
 
 The tag format can be chosen between the following options:
 - **Semantic Versioning:** It matches tags in the popular [SemVer format](https://semver.org/). (e.g. 0.4.2)
-  - `^\\d+.\\d+.\\d+$`
 - **Version contains a prefix:** It matches tags which have an additional prefix before the [SemVer format](https://semver.org/). (e.g. version-0.4.2)
-  - `\\d+.\\d+.\\d+$`
 - **Version contains a suffix:** It matches tags which have an additional suffix after the [SemVer format](https://semver.org/). (e.g. 0.4.2-alpha)
-  - `^\\d+.\\d+.\\d+`
 - **Custom Regular Expression:** You can define your own regex to be matched against tags.
 
+-> **Note:** When managing your workspace with the [hashicorp/tfe provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/workspace#tags_regex) and triggering runs through matching git tags is set, 
+you will need to include an addtional `\` to escape the regex pattern. For example, when you want to trigger a run when git tag is published with semantic versioning: `tags_regex = "^\\d+.\\d+.\\d+$"`. 
+Read more about Terraform [escape sequences](https://www.terraform.io/language/expressions/strings#escape-sequences).
+
+
+| Tag Format                    | Regex Pattern             | Regex Pattern (Escaped)  |
+|-------------------------------|---------------------------|--------------------------|
+| **Semantic Versioning**       | `^\d+.\d+.\d+$`           | `^\\d+.\\d+.\\d+$`       |
+| **Version contains a prefix** | `\d+.\d+.\d+$`            | `\\d+.\\d+.\\d+$`        |
+| **Version contains a suffix** | `^\d+.\d+.\d+`            | `^\\d+.\\d+.\\d+`        |
+
```

Comment:
I think this can just be part of the regular text. Also! I don't think we need the in-text example because you're providing this nice table! Let me know if you feel strongly we should add it back though :) 

```suggestion
You must include an additional `\` to escape the regex pattern when you manage your workspace with the [hashicorp/tfe provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/workspace#tags_regex) and trigger runs through matching git tags. Refer to [Terraform escape sequences](https://www.terraform.io/language/expressions/strings#escape-sequences) for more details.

| Tag Format                    | Regex Pattern             | Regex Pattern (Escaped)  |
|-------------------------------|---------------------------|--------------------------|
| **Semantic Versioning**       | `^\d+.\d+.\d+$`           | `^\\d+.\\d+.\\d+$`       |
| **Version contains a prefix** | `\d+.\d+.\d+$`            | `\\d+.\\d+.\\d+$`        |
| **Version contains a suffix** | `^\d+.\d+.\d+`            | `^\\d+.\\d+.\\d+`        |

```